index.html

```diff
@@ -2,9 +2,141 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <title>02-Box-Model</title>
+    <link rel="stylesheet" href="./app.css">
   </head>
   <body>
+    <nav class="navbar">
+      <h1 class="nav_bar_title">Popular Github</h1>
+
+      <h4 class="nav_bar_text">Repos</h4>
+
+      <h4 class="nav_bar_text">People</h4>
+    </nav>
+
+    <div class="user-profile_parent">
+      <div class="profile">
+        <img src="http://fillmurray.com/100/100" alt="Dustin Walker" />
+        <h1 class="name">Dustin Walker</h1>
+        <h2 class="user_name">DustinJWalker</h2>
+      </div>
+    <div class="repos_1">
+      <h3 class="frame_work">Ember</h3>
+      <div class="repo_dot"></div>
+      <h3 class="lang-name">JavaScript</h3>
+    </div>
+      <div class="repos_2">
+        <h3 class="frame_work">Ember</h3>
+        <span class="repo_dot"></span>
+        <h3 class="lang-name">JavaScript</h3>
+      </div>
+    </div>
+
+    <div class="user-profile_parent">
+      <div class="profile">
+        <img src="http://placecera.com/100/100" alt="Robert Jackson" />
+        <h1 class="name">Robert Jackson</h1>
+        <h2 class="user_name">rwjblue</h2>
+      </div>
+    <div class="repos_1">
+      <h3 class="frame_work">ember-cli-pretender</h3>
+      <div class="repo_dot"></div>
+      <h3 class="lang-name">JavaScript</h3>
+    </div>
+      <div class="repos_2">
+        <h3 class="frame_work">pivot.js</h3>
+        <span class="repo_dot"></span>
+        <h3 class="lang-name">JavaScript</h3>
+      </div>
+    </div>
+
+    <div class="user-profile_parent">
+      <div class="profile">
+        <img src="http://placecage.com/100/100" alt="Peter Wagenet" />
+        <h1 class="name">Peter Wagenet</h1>
+        <h2 class="user_name">wagenet</h2>
+      </div>
+    <div class="repos_1">
+      <h3 class="frame_work">typed-ember</h3>
+      <div class="repo_dot"></div>
+      <h3 class="lang-name">JavaScript</h3>
+    </div>
+      <div class="repos_2">
+        <h3 class="frame_work">rich_text_demo</h3>
+        <span class="repo_dot"></span>
+        <h3 class="lang-name">JavaScript</h3>
+      </div>
+    </div>
+
+    <div class="user-profile_parent">
+      <div class="profile">
+        <img src="http://fillmurray.com/100/100" alt="Bill" />
+        <h1 class="name">Stefan Penner</h1>
+        <h2 class="user_name">stefanpenner</h2>
+      </div>
+    <div class="repos_1">
+      <h3 class="frame_work">es6-promise</h3>
+      <div class="repo_dot"></div>
+      <h3 class="lang-name">JavaScript</h3>
+    </div>
+      <div class="repos_2">
+        <h3 class="frame_work">ember-app-kit</h3>
+        <span class="repo_dot"></span>
+        <h3 class="lang-name">JavaScript</h3>
+      </div>
+    </div>
+
+    <div class="user-profile_parent">
+      <div class="profile">
+        <img src="http://placecera.com/100/100" alt="Bill" />
```

Comment:
You can get the urls for the different user profile images by right clicking on the picture and then clicking "copy Image URL"